### ISSUE TITLE
Getting Invalid prop `style` supplied to `Icon`.

### ISSUE THREAD

**Issue by sardok** [calintamas/react-native-toast-message#150]
My Toast package version is 1.4.6.

During configuration (`<Toast ref={ref => Toast.setRef(ref)} />`), I receive following error;

```
[Tue Feb 16 2021 23:27:09.100]  ERROR    Warning: Failed prop type: Invalid prop `style` supplied to `Icon`.
    in Icon (at base/index.js:42)
    in BaseToast (at success.js:9)
    in SuccessToast (at src/index.js:18)
    in RCTView (at View.js:34)
    in View (at createAnimatedComponent.js:165)
    in AnimatedComponent (at createAnimatedComponent.js:215)
    in ForwardRef(AnimatedComponentWrapper) (at src/index.js:381)
    in Toast (at App.tsx:48)
```

**Comment by helderppb** [comment]
I'm getting the same error. How can I fix it?

**Comment by calintamas** [comment]
Sounds like a regression due to #147. I'll fix it as soon as I can, thanks for raising it

**Comment by kAVArd** [comment]
Also 1.4.3 version have this issue too

**Comment by helderppb** [comment]
Have you updated the npm? I'm installing using npm and the issue still there.

**Comment by Thomas-Negrault** [comment]
The version 1.4.7 fixed the issue for me, thank you for fixing this so quickly ! 🚀

**Comment by calintamas** [comment]
> Have you updated the npm? I'm installing using npm and the issue still there.

yes, published in v1.4.7
